apps/projects/app/components/Panel/NewProject/RepoSelector.js

```diff
@@ -285,8 +308,8 @@ query {
    repositories(
      first: 100,
      orderBy: {field: UPDATED_AT, direction: DESC}
-     ownerAffiliations: [OWNER, COLLABORATOR, ORGANIZATION_MEMBER],
-     affiliations: [OWNER, COLLABORATOR, ORGANIZATION_MEMBER]) {
+     ownerAffiliations: [OWNER, COLLABORATOR],
```

Comment:
Could you please elaborate on why is this change needed?

Reply:
Oh sorry, forgot to change it back after testing.